timemachine/fe/energy_decomposition.py

```diff
@@ -37,17 +42,26 @@ def get_batch_u_fns(bps: Iterable[BoundPotential], temperature: float = DEFAULT_
     """
     kBT = temperature * BOLTZ
 
+    assert len(pots) == len(params)
     batch_u_fns: List[Batch_u_fn] = []
-    for bp in bps:
+    for p, bp in zip(params, pots):
```

Comment:
nit: should we use a name other than `bp` for unbound potentials?

Reply:
Good catch, will make it more explicit. Do we have a convention like we do for bound potentials?

Reply:
I'm not aware of a convention. I think I've mostly used `pot` or `potential` (or just `p` if the scope is local and no potential for confusion with parameters)

Reply:
Fixed up in all the places I could find: [1eff706](https://github.com/proteneer/timemachine/pull/1287/commits/1eff706be5cba8fe393e0eb2fb333d42c405a3da)